types/node/test/stream.ts

```diff
@@ -192,3 +192,17 @@ function stream_readable_pipe_test() {
     z.close();
     rs.close();
 }
+
+async function readable_from() {
+    const list = [ 1, 2, 3 ];
+    const listPromise = list.map(n => Promise.resolve(n));
```

Comment:
a iterable of promises is not an asyncIterable. You could generate your mock asyncIterable with a generator, like this:
```javascript
const asyncList = (async function *() {yield *[1, 2, 3]})()
```

Reply:
Thank you for the correction, fix amended.